data/human/human missions.txt

```diff
@@ -7454,3 +7454,599 @@ mission "Adelita 5"
+					goto doit
+				`	"No, I won't become an investor."`
+					decline
+			label doit
+			`	After you've signed numerous dotted lines, Howard says, "Alright, we have your payment and preliminary signatures. This particular investment is in partnership with Syndicated Systems, so we'll need to submit this first set of paperwork to their office and then go to <destination> to sign the final documents and conduct a biometric scan at our secure facility. I'll go with you to help finalize the paperwork. Once that is complete, you should immediately start receiving dividend payments."`
+				accept
+	on accept
+		payment -1000000
+	on complete
+		conversation
+			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled out, and after scanning your fingerprint, retinas, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign." He places one more set of documents on the table and taps the stack of papers.`
+			`	"New regulations dictate that we inform you that the proceeds of this bond may be used in high risk activities such as strip mining, clearing endangered habitats, or doing business on worlds with alleged human rights violations. All boilerplate language, of course, but you have to disclose every possibility these days." He shrugs.`
+			`	He then taps the documents the documents again and says, "This is your final decision and set of paperwork. I would also like to disclose that your initial deposit covering the full one million credits is non-refundable. So, as a last formality, would you like to invest in the full investment suite?"`
+			choice
+				`	"Of course."`
+					goto high
+				`	"I'm backing out. I can't ethically invest in this."`
+			`	Howard nods. "I get it, ethics are important and I respect your decision. Unfortunately, the initial deposit is non-refundable, nothing I can do there. Best of luck finding another investment that suits you."`
+				decline
+			label high
+			action
+				"salary: Syndicate Business" = 500
+			`	"Excellent. As a bond owner of Megaparsec Incorporated, you will now receive a daily dividend. I have some work to attend to with our major shareholders, but it was a pleasure doing business with you, Captain."`
+
+# Part 2
+# Best case: 10m, 4100 per day best case out of the negative after ~6.7 years
+# Worst case: 11m, 2000 per day worst case out of the negative after ~15 years (13.7 years, plus a little longer for 10M-->11M)
+# Super Worst case: 11m, 2000 per day worst case out of the negative after over 40 years if you don't pay off the debt
+# Driven by if you were mean to Howard in part 1 and part 2, which of 3 investment options picked, and if you pay off debt ASAP
+
+mission "Syndicate Business 3"
+	name "Visit Howard on <planet>"
+	description "Howard, from Megaparsec Inc., wants to tell you about an investment opportunity."
```

Comment:
Now that I think about it, the player already knows who Howard is. So maybe remove the epithet stating he's from Megaparsec?:
```suggestion
	description "Howard wants to tell you about another investment opportunity."
```
Or not:
```suggestion
	description "Howard, from Megaparsec Inc., wants to tell you about another investment opportunity."
```